test-collateral/generate-policy/src/main.rs

```diff
@@ -327,28 +321,23 @@ information to be produced by the executing WASM binary.",
                     "Specifies whether to use interpretation or JIT execution for the WASM \
 binary.",
                 )
-                .required(true)
-                .default_value(DEFAULT_EXECUTION_STRATEGY),
         )
         .arg(
             Arg::with_name("stdin")
-                .short("si")
                 .long("stdin")
                 .value_name("STDIN")
                 .help("The configuration of the standard input in the form 'path:rights'")
                 .required(false),
         )
         .arg(
             Arg::with_name("stdout")
-                .short("so")
```

Comment:
Can we retain a short flag for each of these command line arguments?  e.g. 'e' for `stderr`, 'I' for `stdin`, and 'o' for `stdout`?

Reply:
Perhaps, though `-e` and `-o` are already in use. The existing use could be changed, but how careful would I have to be about changing the meaning of a flag? I can't be sure that all existing uses are in the repo, or that I'd find them all even if they were.

What I could do is use `-E`, `-I`, `-O` for `stderr`, `stdin`, `stdout`.

Reply:
All existing uses of this are almost certainly in this repository, and any problems will be identified by CI, as `generate_policy` is used to generate the policy files needed for integration and unit tests (see e.g. the `test-collateral` Makefile, which invokes `generate_policy`, renamed to `pgen`).  I think there, though, we use the long flags, rather than short.

`E`, `I`, and `O` seem reasonable short names.

Reply:
I've done that. (Changing `-e` to `-x`, and `-x` to something else, etc., might have been possible, but seems less attractive!)